src/NuGet.CommandLine/Commands/PushCommand.cs

```diff
@@ -48,10 +61,9 @@ public class PushCommand : Command
                 timeout = TimeSpan.FromMinutes(5); // Default to 5 minutes
             }
 
-            source = await GetPushEndpointAsync(source);
-            PushPackage(packagePath, source, apiKey, timeout);
+            PushPackage(packagePath, pushEndpoint, apiKey, timeout);
 
-            if (source.Equals(NuGetConstants.DefaultGalleryServerUrl, StringComparison.OrdinalIgnoreCase))
+            if (pushEndpoint.Equals(NuGetConstants.DefaultGalleryServerUrl, StringComparison.OrdinalIgnoreCase))
```

Comment:
It seems that we will always push the symbols if the user is using the nuget.org source, it feels to as a bug that we are not respecting the arguments